[cnats] build failure

### Operating system

windows

### Compiler

gcc

### Steps to reproduce the behavior

```Shell
Please note that I have discovered the problem through debugging, and the following is a description of the problem and the solution to the problem. Please update it in time.

1.Observed behavior
e.g file cnats-targets-debug.cmake

list(APPEND _cmake_import_check_targets cnats::nats )
list(APPEND _cmake_import_check_files_for_cnats::nats "${_IMPORT_PREFIX}/debug/lib/natsd.lib" "${_IMPORT_PREFIX}/debug/ -->bin/natsd.lib<---" ) --->err<---

2.Expected behavior
list(APPEND _cmake_import_check_targets cnats::nats )
list(APPEND _cmake_import_check_files_for_cnats::nats "${_IMPORT_PREFIX}/debug/lib/natsd.lib" "${_IMPORT_PREFIX}/debug/ -->bin/natsd.dll<---" ) --->success<---

note: such err total two , you can search by [bin/natsd.lib] [lib/natsd.dll] , you will get the err point.

3.Server and client version
vcpkg
{
"name": "cnats",
"version": "3.7.0"
}

4.How to fix this bug
https://github.com/microsoft/vcpkg/blob/master/ports/cnats/portfile.cmake
line 49-51 62  lib/natsd.dll --> lib/natsd.dll

lib/nats.dll --> bin/nats.dll not in the file? I found in <cnats-targets.cmake> file
```


### Failure logs

CMake Error at vcpkg_installed/x64-windows/share/cnats/cnats-targets.cmake:85 (message):
  The imported target "cnats::nats" references the file

     "D:/code/quant/hello/vcpkg_installed/x64-windows/debug/lib/natsd.dll"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "D:/code/quant/hello/vcpkg_installed/x64-windows/share/cnats/cnats-targets.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  vcpkg_installed/x64-windows/share/cnats/cnats-config.cmake:32 (include)
  D:/soft/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package)
  src/md8-pub/CMakeLists.txt:4 (find_package)


### Additional context

_No response_

--- a/ports/cnats/fix_install_path.patch
+++ b/ports/cnats/fix_install_path.patch
@@ -1,0 +1,28 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index b3f376c..8e6d06a 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -60,7 +60,10 @@ if(NATS_BUILD_LIB_SHARED)
+   target_include_directories(nats PUBLIC
+         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+         $<INSTALL_INTERFACE:include>)
+-  install(TARGETS nats EXPORT cnats-targets DESTINATION ${NATS_LIBDIR})
++  install(TARGETS nats EXPORT cnats-targets
++        ARCHIVE DESTINATION lib
++		LIBRARY DESTINATION lib
++		RUNTIME DESTINATION bin)
+   install(EXPORT cnats-targets
+         NAMESPACE cnats::
+         FILE cnats-targets.cmake
+@@ -72,7 +75,10 @@ if(NATS_BUILD_LIB_STATIC)
+   target_include_directories(nats_static PUBLIC
+         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+         $<INSTALL_INTERFACE:include>)
+-  install(TARGETS nats_static EXPORT cnats-targets ARCHIVE DESTINATION ${NATS_LIBDIR})
++  install(TARGETS nats_static EXPORT cnats-targets
++        ARCHIVE DESTINATION lib
++		LIBRARY DESTINATION lib
++		RUNTIME DESTINATION bin)
+   install(EXPORT cnats-targets
+         NAMESPACE cnats::
+         FILE cnats-targets.cmake

--- a/ports/cnats/portfile.cmake
+++ b/ports/cnats/portfile.cmake
@@ -6,6 +6,7 @@ vcpkg_from_github(
     HEAD_REF main
     PATCHES
         fix-sodium-dep.patch
+        fix_install_path.patch
 )
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
@@ -74,5 +75,5 @@ vcpkg_copy_pdbs()
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 
-file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
 

--- a/ports/cnats/vcpkg.json
+++ b/ports/cnats/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "cnats",
   "version": "3.7.0",
+  "port-version": 1,
   "description": "A C client for the NATS messaging system",
   "homepage": "https://github.com/nats-io/nats.c",
   "license": "Apache-2.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1678,7 +1678,7 @@
     },
     "cnats": {
       "baseline": "3.7.0",
-      "port-version": 0
+      "port-version": 1
     },
     "cnl": {
       "baseline": "1.1.7",
@@ -2024,7 +2024,7 @@
       "baseline": "10.1",
       "port-version": 13
     },
-    "cuda-api-wrappers" : {
+    "cuda-api-wrappers": {
       "baseline": "0.6.6",
       "port-version": 0
     },

--- a/versions/c-/cnats.json
+++ b/versions/c-/cnats.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "cce27b1efc5af15e4222a5fdc5502959f24e9d03",
+      "version": "3.7.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "438cfff2ee224f9d414bda9bcf796c13534443ba",
       "version": "3.7.0",
       "port-version": 0

